attestation-manager/Makefile

```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-VERSION := $(shell cat VERSION)
+VERSION_SUFFIX              ?= ""
+VERSION                     ?= $(shell cat ./VERSION)$(VERSION_SUFFIX)
```

Comment:
Would you share example how the version looks? Reason  for asking is, in Verfier "init-wait" container, there is dependency for version provided in helm charts and version embedded in verifier component build.